apps/deploy_monitoring/src/deploy_with_compose.rs

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+#![forbid(unsafe_code)]
+
+use std::path::PathBuf;
+use std::process::{Command, Output};
+
+use slog::{info, Logger};
+use tokio::time::{sleep, Duration};
+
+use crate::image::{DeployMonitoringContainer, Explorer, Sandbox, TezedgeDebugger};
+use crate::node::{OcamlNode, TezedgeNode, OCAML_PORT, TEZEDGE_PORT};
+
+pub const DEBUGGER_PORT: u16 = 17732;
```

Comment:
@adonagy 
I would put all (at least port) constants to one file/mod (it shoudl be better as configuration, but constants is enough):
OCAML_PORT, TEZEDGE_PORT, DEBUGGER_PORT